src/settings/BehaviorForm.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  injectIntl,
+  intlShape,
+  FormattedMessage,
+} from 'react-intl';
+import {
+  Form,
+} from 'react-final-form';
+
+import {
+  Pane,
+  PaneFooter,
+  Select,
+  Button,
+} from '@folio/stripes/components';
+
+import RowComponent from './RowComponent';
+
+const deletedRecordsSupportSelectValues = (formatMessage) => [
+  {
+    value: 'no',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.deletedRecordsSupport.no' }),
```

Comment:
Could `<FormattedMessage />` component be used here instead of `formatMessage`?

Reply:
In current case we need to use formatMessage because formatMessage will return string (<FormattedMessage /> will return react element).